packages/icons/src/Feedback.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Svg } from './Svg';
```

Comment:
Should <SVG/> be part of components package? So that others can use it from there without having to import all the icons they don't need? And also, svg is a very genral component, it can be used for more than just icons, I guess.

Reply:
Yes true. Misconception.